docs/contributors/opentracingprovider.adoc

```diff
@@ -177,6 +177,18 @@ The value should be used to set the componentType field on a `Component` node.
 
 If the transaction has not been defined in the trace context, then the value of this tag will be used to set it. This value will then be propagated to other subsequent trace contexts.
 
+==== Tag ‘service’
+
+A free-format value that ties this request to a specific service. When the application is executed in OpenShift, this defaults to a value that is derived from the environment variable `OPENSHIFT_BUILD_NAME`,
```

Comment:
Saw it is listed in the table below - but may be best to also describe here due to the dependency between the env vars if both set.